store/remote_test.go

```diff
@@ -46,19 +48,26 @@ func TestNewRemote(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error reading index: %v", err)
 	}
+
 	if !ok {
 		t.Fatalf("unexpected index not found")
 	}
+
 	if nb.BlobKey != data {
 		t.Fatalf("bad data returned from store: got %v, want %v", nb.BlobKey, data)
 	}
 
 	// Get a remote with a different URI
 	nc, ok, err := s.GetRemote(u2)
-	// Should get an error, since the URI shouldn't be present in the table
+	if err == nil {
+		// Should get an error, since the URI shouldn't be present in the table
+		t.Fatalf("should have received error")
```

Comment:
If you're refering to my answer to @alban, he mentioned that the build didn't pass when I put the checks for `err`, so I asked whether I was the one introducing this bug or this was an uncatched bug.

I didn't write the conditionals like you suggested, because I assumed that there was a reason that the function both has `ok` and `err` as its return values.